crates/astria-sequencer-client/src/extension_trait.rs

```diff
@@ -405,6 +477,32 @@ pub trait SequencerClientExt: Client {
         self.get_nonce(address, 0).await
     }
 
+    /// Get the latest sequencer block.
+    ///
+    /// This is a convenience method that converts the result [`Client::latest_block`]
+    /// to `SequencerBlockData`.
+    async fn latest_sequencer_block(&self) -> Result<SequencerBlockData, Error> {
+        let rsp = self
+            .latest_block()
+            .await
+            .map_err(|e| Error::tendermint_rpc("latest_block", e))?;
+        SequencerBlockData::from_tendermint_block(rsp.block)
+            .map_err(|e| Error::deserialization("SequencerBlockData", e))
+    }
+
+    /// Get the sequencer block at the provided height.
+    ///
+    /// This is a convenience method that converts the result [`Client::block`]
+    /// to `SequencerBlockData`.
+    async fn sequencer_block(&self, height: u32) -> Result<SequencerBlockData, Error> {
```

Comment:
```suggestion
    async fn sequencer_block_data(&self, height: u32) -> Result<SequencerBlockData, Error> {
```

Reply:
I want to replace `SequencerBlockData` by just `SequencerBlock` in the short term, which is the nomenclature we originally introduced in our sequencer protos but never implemented.